### ISSUE TITLE
Refine type information on createElement()'s returns

### ISSUE THREAD

**Issue by psam44** [google/closure-library#570]
This is a continuation on https://github.com/google/closure-library/pull/503, related to https://github.com/google/closure-compiler/issues/1054.

In base.js:
'Element' is too broad as type and prevents disambiguation on property names.

A fix is:
line 1130
change

```
var scriptEl = doc.createElement('script');
```

to

```
var scriptEl = /** @type {HTMLScriptElement} */ (doc.createElement('script'));
```

line 1922
change

```
  var scriptElt = doc.createElement('SCRIPT');
```

to

```
  var scriptElt = /** @type {HTMLScriptElement} */ (doc.createElement('SCRIPT'));
```

Note: A cast is used because a simple type declaration leaves a `WARNING - initializing variable` when compiled with `--warning_level=VERBOSE`.

**Comment by shicks** [comment]
Would you be up for putting together a pull request for this?

**Comment by psam44** [comment]
I don't have the repository to make a PR. The fix is only two tiny copy-paste.

**Comment by shicks** [comment]
You can use GitHub's button to fork this repository.  It may only be a few small changes, but a commit needs to be made, and merged, etc.  Since it's a low-priority issue, we would prefer if you could make that change and then we can just pull it in after Travis tests it.

**Comment by psam44** [comment]
PR done. There is a Travis fail, but it isn't on my changes.

**Comment by bowbahdoe** [comment]
This issue should be closed, a PR was merged that fixed it as mentioned in #584
